lib/views/feedbacks_slack_view.ex

```diff
@@ -12,9 +12,13 @@ defmodule Aprb.Views.FeedbacksSlackView do
     ":artsy-email: #{emoji(message)}"
   end
 
+  def obfuscate_emails(message) do
+    String.replace(message, ~r/(\S+)@\S+/m, "\\1[@domain]", global: true)
```

Comment:
A naive regex for emails, but sufficient for our purposes here, I think: take a run of non-whitespace characters with an `@` in the middle, and obfuscate everything after the `@`.